### ISSUE TITLE
Sync issues with attachments on IE 10

### ISSUE THREAD

**Issue by Wvhf** [nolanlawson/relational-pouch#59]
Hello,

I am building an Ionic app using this plugin to store my data.
I came across a problem when replicating an attachment, containing a base64string of a picture.
First, i add an attachment to the pouchdb, then, i try to replicate the attachment to a remote CloudantDB which returns an error.

In andoid or IOS, everything works fine, but when i try to replicate an attachment on an IE10 device, an error occurs.

The only logging i am getting is the following:
`@dbService | syncData | Error while replicating data to Remote Db`
`ERROR: Object doesn't support property or method 'readAsBinaryString'`

I execute following code after i saved my attachment:
`            Db.getDb().replicate.to(Db.getRemoteDb()).on('complete', function (result) {
                console.log('@dbService | syncData | Data replicated to Remote Db');
            }).on('error', function (error) {
                console.log('@dbService | syncData | Error while replicating data to Remote Db');
                console.log('ERROR: ' + error);
            });`

I also retrieve the following logging when i try to replicate the attachment but I totally dont understand how the filereader plugin might possibly have anything to do with this issue.
`Exception was thrown at line 267, column 9 in ms-appx://be.axi.sramvp1/www/plugins/cordova-plugin-file/www/FileReader.js
0x800a01b6 - JavaScript runtime error: Object doesn't support property or method 'readAsBinaryString'`

To summarize, saving the attachment to the pouchdb just works fine. Only **replicating** to the remote cloudantDB doesn't work.

If anyone else came across this issue, please let me know.

Thanks in advance

**Comment by nolanlawson** [comment]
This error is not in relational-pouch; as shown in your stacktrace it's in cordova-plugin-file. IE10 doesn't have `readAsBinaryString`, so you have to use `readAsArrayBuffer` instead.

**Comment by Wvhf** [comment]
Yes you are correct. After removing the cordova file plugin, everything was working as expected. Somehow pouchdb and the cordova file plugin are in conflict when using both. 

Thanks for pushing me in the right direction.
